build/syntax-highlight.js

```diff
@@ -80,7 +80,10 @@ function syntaxHighlight($, doc) {
     const code = $pre.text();
     const html = Prism.highlight(code, grammar, name);
     const $code = $("<code>").html(html);
+    const wrap = $("<div class='code-example'></div>");
+
     $pre.empty().append($code);
+    $pre.wrapAll(wrap);
```

Comment:
If we can make `$code` be `<div class="code-example"><code>` instead, (and a new name of the variable!) we won't need this, right?

Reply:
Correct!

Reply:
Inevitably we will have to use `.wrapAll()` to wrap each `<pre class="brush: ...">` tag in a `<div class="code-example">` tag. 

Adding .wrapAll to the first `$pre` initialization fixes the HTML structure and the problem of renaming `$code` to `wrap` along with removing the extra line in `$pre.wrapAll()` at the bottom.

```js
$("pre[class*=brush]").each((_, element) => {
    // The language is whatever string comes after the `brush(:)`
    // portion of the class name.
    const $pre = $(element).wrapAll("<div class='code-example'>");
    ....
    const code = $pre.text();
    const html = Prism.highlight(code, grammar, name);
    const wrap = $("<code>").html(html);

    $pre.empty().append(wrap);
```

Reply:
@peterbe Now that I look back at the above code snippet, having `$code` sort of makes more sense then `wrap` if it only is `<code><span>...</code>` since the `<pre>` tags are the ones that are getting wrapped with `<div class='code-example'>`